### ISSUE TITLE
shell module environment does not extend to subsequent commands

### ISSUE THREAD

**Issue by retrogradeorbit** [epiccastle/spire#45]
`(shell :cmd "echo 1; echo $FOO" :env {:FOO "2"})`

FOO is not passed through to second echo command

**Comment by retrogradeorbit** [comment]
fixed in 0.1.0-alpha.7

**Comment by retrogradeorbit** [comment]
```
crispin@vash:~/dev/epiccastle/spire$ spire -e '(shell {:cmd "echo 1; echo $FOO" :env {:FOO "2"}})'
:1 (shell {:env {:FOO "2"}, :cmd "echo 1; echo $FOO"}) local                                                                                       
{:err "", :exit 0, :out "1\n2\n", :out-lines ["1" "2"], :result :ok}
crispin@vash:~/dev/epiccastle/spire$ spire -v
Version: 0.1.0-alpha.7
```
